src/packages/assessment-validator/index.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Is there a reason this needs to be separate from the `packages/validator` directory? It took me a while to understand the build flow because I was confused by there not being any code in `packages/validator`, and the connection to this script isn't obvious

Reply:
I decided to do it this way to follow the `accessibility-insights-ui` package pattern, but yeah, if you think it's cleaner I don't see any blockers to moving this to same directory as the build logic.

Reply:
Looking at this again, I think the ui package code is in its existing location so that it can easily reference/ reuse the code in the rest of the repo. Since we need to do the same thing with the validator package, I think the index file will need to stay in this location.